src/Driver/Connection.php

```diff
@@ -37,7 +37,7 @@ public function query(string $sql): Result;
      *
      * @return mixed
      */
-    public function quote($value, $type = ParameterType::STRING);
+    public function quote($value, int $type = ParameterType::STRING);
```

Comment:
Sorry, please don't keep pushing arbitrary changes this PR. This is really hard to review.

For this change: We have a doc block that documents the parameter as `int`. That should be sufficient for static analysis. The native type is going to be added in 4.0. Please revert all native types added in this PR.

Reply:
Sorry, I will revert this change.